schema.sql

```diff
@@ -4,8 +4,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `password` char(40) NOT NULL,
   `secret` char(16) DEFAULT NULL,
   `type` int(11) NOT NULL DEFAULT '1',
-  `premdays` int(11) NOT NULL DEFAULT '0',
-  `lastday` int(10) unsigned NOT NULL DEFAULT '0',
+  `premend` int(11) NOT NULL DEFAULT '0',
```

Comment:
I'd favor a more verbose `premium_ends_at`, I don't know why we make it so short and vague.